openmc/data/endf_utils.py

```diff
@@ -0,0 +1,45 @@
+"""This module contains a few utility functions for reading ENDF_ data. It is by
+no means enough to read an entire ENDF file.  For a more complete ENDF reader,
+see Pyne_.
+
+.. _ENDF: http://www.nndc.bnl.gov/endf
+.. _Pyne: http://www.pyne.io
+
+"""
+
+import re
+
+def read_float(float_string):
+    """Parse ENDF 6E11.0 formatted string into a float."""
+    assert len(float_string) == 11
+    pattern = '([\s\\-]\d+\\.\d+)([\\+\\-]\d+)'
```

Comment:
Use a raw string for regular expressions. Also, ENDF files are not guaranteed to (and in fact often do not) use exponential form.